packages/pwa-kit-dev/package.json

```diff
@@ -110,6 +110,7 @@
   },
   "devDependencies": {
     "@loadable/component": "^5.15.0",
+    "fs-extra": "^10.1.0",
```

Comment:
We've 3 different versions of `fs-extra` in the monorepo. Do we want to align and use the same latest version? 

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/e997bcad877af44e63854e004e9d652069a924a9/packages/pwa-kit-runtime/package.json#L57

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/e997bcad877af44e63854e004e9d652069a924a9/packages/pwa-kit-create-app/package.json#L35

No new 3PP approval is required. We already have approved v10.1.0.

Tip: You can use `npx lerna-update-wizard`